metaflow/plugins/__init__.py

```diff
@@ -137,28 +139,30 @@ def get_plugin_cli():
 AWS_CLIENT_PROVIDERS = resolve_plugins("aws_client_provider")
 SECRETS_PROVIDERS = resolve_plugins("secrets_provider")
 
+from .argo.argo_events import ArgoEvent
```

Comment:
I am iterating on the ArgoEvent UX, but this would be exposed to the user so that they can directly invoke the client to emit events either from within a flow or a different Python process.